syntax = "proto2";

enum EMessageType
{
	ENone = 0;

	ESCFrameInit = 100;
	ECSFrame = 101;
	ESCFrame = 102;

	ECSLogin = 10001;
	ESCLogin = 10002;

	ECSReady = 11001;
	ESCReady = 11002;

	ESCSight = 20001;

	EMax = 999999;
}
